test/scripts/MVC4/login.test.js

```diff
@@ -78,7 +78,7 @@ describe(scriptName, () => {
     row.mockImplementation((callback) => callback({
       UserId: { value: 'uid1' },
       UserName: { value: 'duck.t@example.com' },
-      Password: { value: 'password' }
+      Password: { value: 'AINcmtaNXJGRJblFbwak2MxyDWwzMKx5uK+p/Vq+fJ6XKCuQARaWP6TU31VsHF2+ZA==' }
```

Comment:
The `crypto` api has stricter enforcement of inputs in modern node versions, so this was [failing](https://github.com/auth0/db-connections-templates/actions/runs/16526417747/job/46740786406#step:5:96) in node22